python-checks/src/main/java/org/sonar/python/checks/UselessParenthesisCheck.java

```diff
@@ -19,93 +19,68 @@
  */
 package org.sonar.python.checks;
 
-import com.sonar.sslr.api.AstNode;
 import java.util.List;
+
 import org.sonar.check.Priority;
 import org.sonar.check.Rule;
 import org.sonar.python.PythonCheck;
 import org.sonar.python.api.PythonGrammar;
+import org.sonar.python.api.PythonKeyword;
 import org.sonar.python.api.PythonPunctuator;
 import org.sonar.squidbridge.annotations.ActivatedByDefault;
 import org.sonar.squidbridge.annotations.SqaleConstantRemediation;
-import org.sonar.sslr.ast.AstSelect;
+
+import com.sonar.sslr.api.AstNode;
 
 @Rule(
     key = UselessParenthesisCheck.CHECK_KEY,
     priority = Priority.MAJOR,
-    name = "Useless parentheses around expressions should be removed to prevent any misunderstanding",
+    name = "Redundant parentheses around expressions should be removed",
     tags = Tags.CONFUSING
 )
 @SqaleConstantRemediation("1min")
 @ActivatedByDefault
 public class UselessParenthesisCheck extends PythonCheck {
+
   public static final String CHECK_KEY = "S1110";
 
+  private static final String MESSAGE = "Remove those useless parentheses.";
+
   @Override
   public void init() {
     subscribeTo(
```

Comment:
Unit tests are still green if we replace this whole list with just ATOM and update visitNode accordingly.